cluster/charts/crossplane/crds/cache/v1alpha1/redisclusterpolicy.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
do we need to make an update to the examples or docs in this PR too, or are we tracking that work separately in #594 and #595?

Reply:
Correct! Tracking separately in #594 and #595. PR's for both of those should land around the same time so that docs will not be out of date.